src/os/bluestore/bluestore_types.h

```diff
@@ -667,7 +667,7 @@ struct bluestore_onode_t {
   uint64_t nid = 0;                    ///< numeric id (locally unique)
   uint64_t size = 0;                   ///< object size
   map<string, bufferptr> attrs;        ///< attrs
-  uint64_t omap_head = 0;              ///< id for omap root node
+  uint8_t omap_head_exist = 0;         ///< true if omap truly used
```

Comment:
how about uint8_t flags = 0;
and
enum {
  FLAG_OMAP = 1;
};
and teh usual test_flag, set_flag, clear_flag helpers, plus a has_omap() helper?